hadoop-client-modules/hadoop-client-integration-tests/pom.xml

```diff
@@ -191,6 +191,12 @@
           <scope>test</scope>
           <type>test-jar</type>
         </dependency>
+        <dependency>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-common</artifactId>
+          <scope>test</scope>
+          <type>test-jar</type>
```

Comment:
nit: actually we may merge this with line 173

Reply:
This is type test-jar. Can we merge with type jar (line 173)?

Reply:
It worked as far as I tried on #3447.

Reply:
Yes it looks fine to me too.